cmd/juju/cloud/addcredential_test.go

```diff
@@ -891,7 +891,7 @@ Enter credential name:
 Using auth-type "jsonfile".
 
 Enter path to the .json file containing a service account key for your project
-(detailed instructions available at https://discourse.jujucharms.com/t/1508).
+(detailed instructions available at https://discourse.charmhub.io/t/1508).
```

Comment:
absolutely better than not to update this URL.
That said, I don't feel that our official/linked documentation for how to add google cloud platform credentials should be at that URL. Maybe something like "juju.is/docs/add-google-cloud-platform" with a redirect, etc. I'll bring that up independently with Dora.